Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/LandscapeIntegrationTest.java

```diff
@@ -424,6 +439,7 @@ public void testFullscreenPortraitOrientationChangeWithShape() {
 		onToolBarView()
 				.performOpenToolOptionsView()
 				.performCloseToolOptionsView();
+		IdlingRegistry.getInstance().unregister(idlingResource);
```

Comment:
You already invoked .unregister() in the tearDown method, so I think there is no need to additionally .register() at the start of the test.